zenith_local.md

```diff
@@ -0,0 +1,103 @@
+# Zenith local
+
+Here I list some objectives to keep in mind when discussing zenith-local design and a proposal that brings all components together.  Your comments on both parts are very welcome.
+
+#### Why do we need it?
+- For distribution - this easy to use binary will help us to build adoption among developers.
+- For internal use - to test all components together.
+
+In my understanding, we consider it to be just a mock-up version of zenith-cloud.
+> Question: How much should we care about durability and security issues for a local setup?
+
+
+#### Why is it better than a simple local postgres?
+
+- Easy one-line setup. As simple as `cargo install zenith && zenith start`
```

Comment:
Actually that probably wouldn't -- I researched topic. Cargo deliberately prohibits a lot actions needed for package manager like pre/post scripts, included files, etc. If software isn't completely in rust (and we have to carry at least postgres) it is hard to use cargo.

Reply:
So we may use brew (mac), choco (win), snap (linux)

Reply:
We can use a Makefile and `make install`. It can call `cargo` for all the Rusty parts.